test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/providers/ops/schedule/RandomSchedule.java

```diff
@@ -0,0 +1,126 @@
+import com.hedera.services.bdd.spec.infrastructure.OpProvider;
+import com.hedera.services.bdd.spec.infrastructure.providers.LookupUtils;
+import com.hedera.services.bdd.spec.infrastructure.providers.names.RegistrySourcedNameProvider;
+import com.hedera.services.bdd.spec.transactions.schedule.HapiScheduleCreate;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import com.hederahashgraph.api.proto.java.ScheduleID;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.hedera.services.bdd.spec.infrastructure.providers.ops.crypto.RandomAccount.INITIAL_BALANCE;
+import static com.hedera.services.bdd.spec.infrastructure.providers.ops.crypto.RandomTransfer.stableAccounts;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoTransfer;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.scheduleCreate;
+import static com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoTransfer.tinyBarsFromTo;
+import static com.hedera.services.bdd.suites.HapiApiSuite.A_HUNDRED_HBARS;
+import static com.hedera.services.bdd.suites.HapiApiSuite.DEFAULT_PAYER;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_DELETED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INSUFFICIENT_ACCOUNT_BALANCE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.MEMO_TOO_LONG;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNRESOLVABLE_REQUIRED_SIGNERS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNSCHEDULABLE_TRANSACTION;
+import static java.util.stream.Collectors.toList;
+
+public class RandomSchedule implements OpProvider {
+	private final AtomicInteger opNo = new AtomicInteger();
+	private final RegistrySourcedNameProvider<ScheduleID> schedules;
+	private final EntityNameProvider<AccountID> accounts;
+	public final ResponseCodeEnum[] permissibleOutcomes = standardOutcomesAnd(MEMO_TOO_LONG,
+			UNSCHEDULABLE_TRANSACTION,
+			UNRESOLVABLE_REQUIRED_SIGNERS);
+
+	private final ResponseCodeEnum[] outcomesForTransfer = standardOutcomesAnd(
+			ACCOUNT_DELETED,
+			INSUFFICIENT_ACCOUNT_BALANCE
+	);
+
+	public static final int DEFAULT_CEILING_NUM = 100;
+	private int ceilingNum = DEFAULT_CEILING_NUM;
+	static final String ADMIN_KEY = DEFAULT_PAYER;
+	static final String STABLE_RECEIVER = "stable-receiver";
+
+	public RandomSchedule(
+			RegistrySourcedNameProvider<ScheduleID> schedules,
+			EntityNameProvider<AccountID> accounts
+	) {
+		this.schedules = schedules;
+		this.accounts = accounts;
+	}
+
+	public RandomSchedule ceiling(int n) {
+		ceilingNum = n;
+		return this;
+	}
+
+	@Override
+	public List<HapiSpecOperation> suggestedInitializers() {
+		return stableAccounts(1).stream()
```

Comment:
Are we scheduling all transfers go one account?

Reply:
@ljianghedera  Yes that is correct. This is the simple way to generate random schedule transactions, in which one signer (`receiver`) is needed.

Reply:
Ok.